gzip.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/gin-gonic/gin"
+	"gopkg.in/gin-gonic/gin.v1"
```

Comment:
We are moving everything back to `github.com/gin-gonic/gin` in the time of dependency managers.